parachains/integration-tests/statemine/xcm/3_hrmp-open-channels.yml

```diff
@@ -170,245 +77,76 @@ tests:
           - name: Penpal Parachain sends a request to the Relay Chain to open a channel with the Assets Parachain
             actions:
               - extrinsics:
-                - chain: *penpal_parachain
-                  signer: *pp_signer
+                - chain: *relay_chain
+                  signer: *rc_signer
                   sudo: true
-                  pallet: polkadotXcm
-                  call: send
+                  pallet: hrmp
+                  call: forceOpenHrmpChannel
                   args: [
-                    *rc_dest, # destination
-                    {
-                      v3: [ #message
-                        {
-                          WithdrawAsset: [*rc_ksm_fungible]
-                        },
-                        {
-                          BuyExecution: {
-                              fees: *rc_ksm_fungible,
-                              weightLimit: Unlimited
-                          }
-                        },
+                    2000,
```

Comment:
I guess this has to be the opposite for this test [1000, 2000...]
I have fixed it in integration-tests repo.